README.md

```diff
@@ -1,13 +1,24 @@
-# Kubeflow Anywhere
+# Kubeflow
 
-The Kubeflow project is a new open source Github repo dedicated to reducing friction in doing Machine Learning on Kubernetes. Contained in this repository are:
+The Kubeflow project is dedicated to making Machine Learning on Kubernetes easy, portable and scalable. Contained in this repository are:
 
 * JupyterHub to create & manage interactive Jupyter notebooks
 * A Tensorflow Custom Resource (CRD) that can be configured to use CPUs or GPUs, and adjusted to the size of a cluster with a single setting
 * A Tensorboard instance
 * A TF Serving container
 
-This document details the steps needed to run the kubeflow project in different environments, such as Minikube (local laptop). Google Kubernetes Engine, etc. 
+This document details the steps needed to run the kubeflow project in any environment in which Kubernetes runs.
+
+## The Kubeflow Mission
+
+Our goal is to help folks use ML more easily, by letting Kubernetes to do what it's great at:
+- Easy, repeatable, portable deployments
+- Deploying and managing loosely-coupled microservices
+- Scaling based on demand
+
+Because ML practitioners use so many different types of tools, it is a key goal that you can customize the stack to whatever your requirements (within reason), and let the system take care of the "boring stuff." While we have started with a narrow set of technologies, we are working with many different projects to include additional tooling.
+
+Ultimately, we want to have a single deployable artifact (a manifest) that gives you an easy to use ML stack _anywhere_ Kubernetes is already running and can self configure based on the cluster it deploys into.
```

Comment:
a set of deployable manifests? Because we're doing a directory of them, with the application step being:
`kubectl apply -f dir/ -R

Reply:
great point, fixed!